### PR TITLE
Page titles

### PR DESCRIPTION
**Pull request by fededirocco**
## Summary
* Implemented title in Index 
* Modified scss for title
* Added folder img and image logo
* Added colors and other styles (in setup project description)

## Trello Card
https://trello.com/c/tRYzvtaz/21-page-titles

## Screenshots
page-titles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="scss/application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <div class="title">
+    <img src="img/triangle.svg" />
+    <p>Front-End Tools</p>
```
Comment (jjuli93) [inline index.html]: This should be a title (h1 or h2)

File: scss/styles.scss [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+body {
```
Comment (jjuli93) [inline scss/styles.scss]: do not use empty rulesets

File: scss/titles.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.title {
+  align-items: center;
+  display: flex;
+  margin: 10px;
+  position: relative;
+
+  p {
+    color: $black;
+    font-family: 'Roboto', sans-serif;
```
Comment (jjuli93) [inline scss/titles.scss]: the font should be the same for the whole document

File: index.html [hunk 4]
```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="scss/application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <div class="title">
+    <img src="img/triangle.svg" />
```
Comment (jjuli93) [inline index.html]: It would be cleaner to declare the image in the css as a before pseudo element

File: scss/titles.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
+.title {
+  align-items: center;
+  display: flex;
+  margin: 10px;
+  position: relative;
+
+  p {
```
Comment (jjuli93) [inline scss/titles.scss]: It is good practice to reference a class instead of referencing an html element

File: scss/titles.scss [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    color: $black;
+    font-family: 'Roboto', sans-serif;
+    font-size: 40px;
```
Comment (jjuli93) [inline scss/titles.scss]: you should use the sizes defined in styles.scss ($text-big, $text-medium, $text-small)

File: scss/styles.scss (hunk 2 above)
Comment (FrankIglesias) [reply scss/styles.scss]: And avoid as much as posible the ussage of tags in scss or css

File: scss/titles.scss [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    left: 85px;
+    position: absolute;
```
Comment (FrankIglesias) [inline scss/titles.scss]: Why is the text in absoute?

File: index.html [hunk 8]
```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet"  type="text/css" href="scss/application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <div class="title">
```
Comment (FrankIglesias) [inline index.html]: This div is not necessary

File: scss/titles.scss [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  width: 100%;  
+}
+
+.title-name {
+  bottom: 28px;
+  color: $black;
+  font-size: $text-big;
+  left: 50px;
+  position: absolute;
+}
```
Comment (FrankIglesias) [inline scss/titles.scss]: add empty line at the end of the file

File: scss/styles.scss [hunk 10]
```diff
@@ -1,3 +1,14 @@
...
+    font-family: 'Roboto', sans-serif;
+}
```
Comment (FrankIglesias) [inline scss/styles.scss]: same

File: scss/titles.scss (hunk 7 above)
Comment (FrankIglesias) [reply scss/titles.scss]: You can set this absolute only to the pseudo element

**Pull request merged**